js/menu.js

```diff
@@ -0,0 +1,18 @@
+// Dropdown Menu
+const btncls = document.querySelector('#closeOnClick');
+const btnopn = document.querySelector('#openOnClick');
+const menuItem = document.querySelectorAll('#menuOpt li a');
+
+const btnOpnNav = () => {
+  document.getElementById('myNav').style.height = '100%';
+};
+
+const btncloseNav = () => {
+  document.getElementById('myNav').style.height = '0%';
+};
```

Comment:
- [ OPTIONAL] This a good Javascript Best Practice but we can make it DRY and easy to read for being more professional, the `setNavHeight` function sets the height of the `myNav` element, and the `btnOpnNav` and `btncloseNav` functions call this function to set the height accordingly. By doing this, you eliminate the duplication of code and make the code more maintainable and easier to read. 
For example if you do something like this :point_down: 
```const btnOpnNav = () => {
  setNavHeight('100%');
};

const btncloseNav = () => {
  setNavHeight('0%');
};

Then these event listeners below allow the user to interact with the dropdown menu by clicking on the buttons and menu items. When the buttons and menu items are clicked, the corresponding functions are executed, which update the height of the `myNav` element to either show or hide the dropdown menu. :wink:  try to make your codes DRY for being more professional :wink: 
`btnopn.addEventListener('click', btnOpnNav);
btncls.addEventListener('click', btncloseNav);
menuItem.forEach((item) => {
  item.addEventListener('click', btncloseNav);
});`

Reply:
Thank you very much @Alfredbis29 , I appreciate your advice and recommendations to improve my code, I will make the changes you suggest. Thanks for your time.